Static/main.css

```diff
@@ -1,3 +1,54 @@
 body {
     background-color: gray;
+}
+
+.topnav {
+    background-color: #091C66;
+    overflow: hidden;
+    position: sticky;
+    height: 70px;
+    display: block;
+}
+
+.topnav > a {
+    text-decoration: none;
+    color: white;
+}
+
+.right-side {
+    position: relative;
+    padding: 5px;
+    margin: 5px;
+    margin-right: 20px;
+    float: right;
+    display: block;
+    margin-top: 15px;
+}
+.left-side {
+    position: relative;
+    padding: 5px;
+    margin: 5px;
+    margin-right: 20px;
+    margin-top: 0px;
+    float: left;
+    display: block;
+}
+
+#logout {
+    width: 30px;
```

Comment:
In principe zou je hiervoor ook .leftside > img kunnen doen maar dit is ook prima